### PR TITLE
Feature/add pages

### PR DESCRIPTION
**Pull request by AlexandrMasalitin**

### CODE REVIEW

**Comment by AlexandrMasalitin** [comment]
Adding pages and routing for them

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,10 +2,20 @@ import { NgModule } from '@angular/core';
...
 import { HomeComponent } from './pages/home-page/home.component';
+import { CategoriesComponent } from './pages/categories-page/categories.component';
+import { PlaylistComponent } from './pages/playlist-page/playlist.component';
+import { RegistrationComponent } from './pages/registration-page/registration.component';
+import { ReleasesComponent } from './pages/releases-page/releases.component';
+import { SelectedCategoriesComponent } from './pages/selected-categories-page/selected-categories.component';
 
 export const routes: Routes = [
   {path: '', component: HomeComponent, pathMatch: 'full'},
-  {path: '**', component: HomeComponent}
```
Comment (Demavend) [inline src/app/app-routing.module.ts:8]: Why this line was removed?

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,2 +1,17 @@
-<router-outlet></router-outlet>
+<ul>
+  <li class="active"><a routerLink="/">Главная</a></li>
```
Comment (Demavend) [inline src/app/app.component.html:2]: Please use only one language in the project - Russian or English (better English). Try to use ngFor to add all links.

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+  <li><a routerLink="/categories">Категории</a></li>
+  <li><a routerLink="/playlist">Плейлист</a></li>
+  <li><a routerLink="/registration">Регистрация</a></li>
```
Comment (Demavend) [inline src/app/app.component.html:5]: I asked to rename the "registration" to "authorization" or "authentication".

File: src/app/app.module.ts [hunk 4]
```diff
@@ -1,18 +1,29 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { HttpClientModule } from '@angular/common/http';
-import { AppRoutingModule } from './app-routing.module';
+import {AppRoutingModule, routes} from './app-routing.module';
```
Comment (Demavend) [inline src/app/app.module.ts:3]: Please configure your IDE and restore the spaces inside the curly braces.

**Pull request merged**